helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixAdmin.java

```diff
@@ -731,6 +731,36 @@ public boolean isReadyForPreparingJoiningCluster(String clusterName, String inst
     return false;
   }
 
+  @Override
+  public boolean forceKillInstance(String clusterName, String instanceName) {
+    return forceKillInstance(clusterName, instanceName, "Force kill instance", null);
```

Comment:
forceKillInstance method allows users to specify the reason and operationSource of the UNKNOWN instance operation that is set as part of the forceKillInstance logic. If no reason is passed, I figured it would still be a good idea to clarify that the UNKNOWN operation was set by the forceKillInstance path